streams/src/test/java/org/apache/kafka/test/ProcessorTopologyTestDriver.java

```diff
@@ -125,7 +124,7 @@
  * ProducerRecord<String, String> record2 = driver.readOutput("output-topic-1", strDeserializer, strDeserializer);
  * ProducerRecord<String, String> record3 = driver.readOutput("output-topic-2", strDeserializer, strDeserializer);
  * </pre>
- *
+ * <p>
```

Comment:
not required

Reply:
ok. IDEA does this when there's a blank line in the comment. I'll take it out.